### DISCUSSION TITLE
How to show or hide code individually

### DISCUSSION THREAD

**Discussion by eyllanesc** [HEIGE-PCloud/DoIt#314] [General]
In config.toml you can define the number of rows that determine if the code is shown or hidden through the `maxShownLines` attribute:

https://github.com/HEIGE-PCloud/DoIt/blob/dc3b174b5e55f7659ce9dc8c7d8f008b1fdaf6ca/exampleSite/config.toml#L680-L688

The point is that in some cases you want to do it manually and not use the maxShownLines. Is there a way to get that behavior?

**Comment by HEIGE-PCloud** [comment]
Currently there is no way to do that, the same reason why #207 is difficult to implement.

Hugo does not provide hooks for code blocks https://github.com/gohugoio/hugo/issues/6702 so there is no way to pass an `expand` argument like this

````
```toml expand
// ...
```
````

There is a way to get around this using client side JavaScript (see #215) or maybe a custom `code` shortcode can do the trick, but I think neither of them solve the problem elegantly.
